api/v1alpha1/shipwrightbuild_types.go

```diff
@@ -14,8 +14,14 @@ type ShipwrightBuildSpec struct {
 	TargetNamespace string `json:"targetNamespace,omitempty"`
 }
 
-// ShipwrightBuildStatus defines the observed state of Shipwright-Build
-type ShipwrightBuildStatus struct{}
+// ShipwrightBuildStatus defines the observed state of ShipwrightBuild
+type ShipwrightBuildStatus struct {
+	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
```

Comment:
Can you delete the scaffolding code comments?

Reply:
done